tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/detection.py

```diff
@@ -933,3 +895,101 @@ def process(self, raw, identifiers, frame_meta):
             results.append(DetectionPrediction(identifier, labels, scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return results
+
+
+class NanoDetAdapter(Adapter):
+    __provider__ = 'nanodet'
+
+    @classmethod
+    def validate_config(cls, config, fetch_only=False, **kwargs):
+        return super().validate_config(
+            config, fetch_only=fetch_only, on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT
+        )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'num_classes': NumberField(
+                description="Number of classes.", value_type=int, min_value=0, default=80, optional=True),
+            'confidence_threshold': NumberField(optional=True, default=0.05, description="confidence threshold"),
+            'nms_threshold': NumberField(optional=True, default=0.6, description="NMS threshold"),
+            'max_detections': NumberField(optional=True, value_type=int, default=100,
+                                          description="maximal number of detections")
+        })
+        return parameters
+
+    def configure(self):
+        self.num_classes = self.get_value_from_config('num_classes')
+        self.confidence_threshold = self.get_value_from_config('confidence_threshold')
+        self.nms_threshold = self.get_value_from_config('nms_threshold')
+        self.max_detections = self.get_value_from_config('max_detections')
+        # Set default values
+        self.strides = [8, 16, 32]
+        self.reg_max = 7
```

Comment:
are there situation when this value can be changed? Possibly we should make them also configurable, but with default parameters

Reply:
Done